fixtures/attribute-behavior/src/App.js

```diff
@@ -958,35 +958,25 @@ class App extends React.Component {
     return (
       <div>
         <div>
-          <select onChange={this.onUpdateSort}>
-            <option
-              selected={this.state.sortOrder === ALPHABETICAL}
-              value={ALPHABETICAL}>
+          <select value={this.state.sortOrder} onChange={this.onUpdateSort}>
+            <option value={ALPHABETICAL}>
               alphabetical
             </option>
-            <option
-              selected={this.state.sortOrder === REV_ALPHABETICAL}
-              value={REV_ALPHABETICAL}>
+            <option value={REV_ALPHABETICAL}>
               reverse alphabetical
             </option>
-            <option
-              selected={this.state.sortOrder === GROUPED_BY_ROW_PATTERN}
-              value={GROUPED_BY_ROW_PATTERN}>
+            <option value={GROUPED_BY_ROW_PATTERN}>
               grouped by row pattern :)
             </option>
           </select>
-          <select onChange={this.onUpdateFilter}>
-            <option selected={this.state.sortOrder === ALL} value={ALL}>
+          <select value={this.state.sortOrder} onChange={this.onUpdateFilter}>
```

Comment:
I think this is wrong (not your doing). Could you change this to `this.state.filter`? It looks like this dropdown isn't working.

Reply:
@nhunzaker Done, thanks for the reviewing.